src/platforms/php/guides/symfony/index.mdx

```diff
@@ -19,30 +19,46 @@ composer require sentry/sentry-symfony
 
 ### Configure
 
-Add your DSN to `app/config/config.yml`:
+Add your DSN to `config/packages/sentry.yaml`:
 
-```yaml {filename:app/config/config.yml}
+```yaml {filename:config/packages/sentry.yaml}
 sentry:
   dsn: "___PUBLIC_DSN___"
 ```
 
-If you **are not** using Symfony Flex, you'll also need to enable the bundle in `app/AppKernel.php`:
+If you **are not** using Symfony Flex, you'll also need to enable the bundle in `config/bundles.php`:
 
-```php {filename:app/AppKernel.php}
+```php {filename:config/bundles.php}
 <?php
-class AppKernel extends Kernel
-{
-    public function registerBundles()
-    {
-        $bundles = array(
-            // ...
 
-            new Sentry\SentryBundle\SentryBundle(),
-        );
+return [
+    // ...
+    Sentry\SentryBundle\SentryBundle::class => ['all' => true],
+];
+```
 
-        // ...
-    }
+### Monolog Integration
 
-    // ...
-}
+If monolog is used some additional configuration and services is needed
+to log the errors correctly:
+
+```php {filename:config/packages/sentry.yaml}
+sentry:
+    register_error_listener: false # Disables the ErrorListener to avoid duplicated log in sentry
+    dsn: "%env(SENTRY_DSN)%"
+
+monolog:
+    handlers:
+        sentry:
+            type: service
+            id: Sentry\Monolog\Handler
+
+services:
+    Monolog\Processor\PsrLogMessageProcessor:
+        tags: { name: monolog.processor, handler: sentry }
+
+    Sentry\Monolog\Handler:
+        arguments:
+            $hub: '@Sentry\State\HubInterface'
+            $level: !php/const Monolog\Logger::ERROR
 ```
```

Comment:
Can we add a verification step, similar to [this model](https://docs.sentry.io/platforms/javascript/#verify) in JavaScript?

Also, if we add this line to the end of the file, it will automatically list the associated supporting content:

`<PageGrid header="Next Steps" nextSteps />`

Reply:
That is a good idea I think a controller with `$this->logger->error` called would be a good idea here to very if its working.

Reply:
Example added to trigger and error and exception. (Both cases should be tested that the trigger sentry correctly).